RemoteDispatch.csproj

```diff
@@ -1,30 +1,35 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>net48</TargetFramework>
```

Comment:
Please keep .NET Standard 2.0 as this is what Unity officially supports.

Reply:
UMM is built against .NET Framework, which causes compiler warnings when the project's target framework is .NET Standard.